src/tests/ftest/scrubber/target_auto_eviction.py

```diff
@@ -32,10 +34,31 @@ def test_scrubber_ssd_auto_eviction(self):
         self.create_pool_cont_with_scrubber(pool_prop=pool_prop, cont_prop=cont_prop)
         self.dmg_cmd.pool_query(self.pool.identifier)
         initial_metrics = self.scrubber.get_scrub_corrupt_metrics()
+        t_start = journalctl_time()
+        t_end = journalctl_time(datetime.now() + timedelta(seconds=60))
         self.run_ior_and_check_scruber_status(pool=self.pool, cont=self.container)
         # Wait for a minute for the scrubber to take action and evict target
         # after corruption threshold reached.
+        self.log.info("Sleeping for 60 seconds")
         time.sleep(60)
```

Comment:
This 60s could be incorrect if, for example, `run_ior_and_check_scruber_status` takes a long time to run.
I think it's better to use the actual timestamp
```suggestion
        t_start = journalctl_time()
        self.run_ior_and_check_scruber_status(pool=self.pool, cont=self.container)
        # Wait for a minute for the scrubber to take action and evict target
        # after corruption threshold reached.
        self.log.info("Sleeping for 60 seconds")
        time.sleep(60)
        t_end = journalctl_time()
```

Reply:
Agree. Initial code was like the suggestion. Then I got sucked into the journalctl not finding the matching string when I used the get_journalctl method... After fixing the method to use the get_journalctl_command(), it should fine now. 
Changes done. Thanks.